src/inputs/datePickerInput/datePickerInput.jsx

```diff
@@ -617,10 +625,11 @@ class DatePickerInput extends React.PureComponent {
                                         color={iconColor}
                                         compact
                                         disable={isDisabled}
-                                        onClick={this.onIconClick}
+                                        onClick={isRedacted ? null : this.onIconClick}
```

Comment:
NAB: Is `null` a good value here?  Might `undefined` or `noop` be better?
NAB and discretionary if this is working code.
JS is _so confusing_ with both `undefined` and `null` ... sigh.

Reply:
there should be some kind of nil value, not `noop`

Reply:
This is probably fine as-is.  Resolving.